cgmanifest.json

```diff
@@ -1,5 +1,6 @@
 {
-  "Registrations": [
+  "$schema": "https://json.schemastore.org/component-detection-manifest.json",
+  "registrations": [
```

Comment:
Please don't change the casing - I think some of our tooling might start failing.